From acc78aee2fab17fad2a258c70a9b7253d4e4a2d0 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Fri, 13 Nov 2020 08:47:01 +0900
Subject: [PATCH 66/99] ODROID-N2/C4/HC4: arm64/dts: remove property 'resets'
 and 'reset-names'

Having this properties in the node 'ethmac', it does cause to load wrong
MAC address.

Signed-off-by: Dongjin Kim <tobetter@gmail.com>
Change-Id: Ifca39bb60d4cbb6586ef9a8789f6dd243352dc1e
---
 arch/arm64/boot/dts/amlogic/meson64_odroidc4.dts  | 5 +++++
 arch/arm64/boot/dts/amlogic/meson64_odroidhc4.dts | 5 +++++
 arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts  | 5 +++++
 3 files changed, 15 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson64_odroidc4.dts b/arch/arm64/boot/dts/amlogic/meson64_odroidc4.dts
index 64b7645e87c4..b929eee1a160 100644
--- a/arch/arm64/boot/dts/amlogic/meson64_odroidc4.dts
+++ b/arch/arm64/boot/dts/amlogic/meson64_odroidc4.dts
@@ -27,6 +27,11 @@ opp-2100000000 {
 	};
 };
 
+&ethmac {
+	/delete-property/	resets;
+	/delete-property/	reset-names;
+};
+
 &vddcpu {
 	regulator-max-microvolt = <1030000>;
 };
diff --git a/arch/arm64/boot/dts/amlogic/meson64_odroidhc4.dts b/arch/arm64/boot/dts/amlogic/meson64_odroidhc4.dts
index 2579a9e4ccfb..b983cf00da72 100644
--- a/arch/arm64/boot/dts/amlogic/meson64_odroidhc4.dts
+++ b/arch/arm64/boot/dts/amlogic/meson64_odroidhc4.dts
@@ -52,6 +52,11 @@ opp-2100000000 {
 	};
 };
 
+&ethmac {
+	/delete-property/	resets;
+	/delete-property/	reset-names;
+};
+
 &fan0 {
 	fan-supply = <&vcc_5v>;
 	interrupt-parent = <&gpio_intc>;
diff --git a/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts b/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
index 799c8409b371..b541ec1a88f6 100644
--- a/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
@@ -125,6 +125,11 @@ codec {
 	};
 };
 
+&ethmac {
+	/delete-property/	resets;
+	/delete-property/	reset-names;
+};
+
 &spdifout {
 	pinctrl-0 = <&spdif_out_a13_pins>;
 	pinctrl-names = "default";
-- 
2.37.3

